libdeno/deno.h

```diff
@@ -76,7 +76,7 @@ int deno_execute(Deno* d, void* user_data, const char* js_filename,
 // Return value: 0 = fail, 1 = success
 // Get error text with deno_last_exception().
 int deno_execute_mod(Deno* d, void* user_data, const char* js_filename,
-                     const char* js_source);
+                     const char* js_source, bool resolve_only);
```

Comment:
Don't use `bool` here. `deno.h` is supposed to be ANSI C to support linking to Rust. Even if this isn't a problem currently, it is in Go, and we might be conservative.

Add a line of documentation regarding this parameter, please.